mongodbatlas/data_source_mongodbatlas_x509_authentication_database_user.go

```diff
@@ -66,7 +67,8 @@ func dataSourceMongoDBAtlasX509AuthDBUserRead(ctx context.Context, d *schema.Res
 	username := d.Get("username").(string)
 
 	if username != "" {
-		certificates, _, err := conn.X509AuthDBUsers.GetUserCertificates(ctx, projectID, username)
+		options := &matlas.ListOptions{}
+		certificates, _, err := conn.X509AuthDBUsers.GetUserCertificates(ctx, projectID, username, options)
```

Comment:
[nit] you don't need to pass an empty option here, `nil` is perfectly fine 
```suggestion
		certificates, _, err := conn.X509AuthDBUsers.GetUserCertificates(ctx, projectID, username, nil)
```

Reply:
I do have this pattern in resources/tests as well I have removed